spec/models/user_spec.rb

```diff
@@ -18,7 +18,19 @@
 
 RSpec.describe User, type: :model do
   it "is valid when produced by a factory" do
-    user = FactoryBot.build(:user)
-    expect(user).to be_valid
+	user = FactoryBot.build(:user)
+	expect(user).to be_valid
+  end
+
+  it 'is valid with image as avatar' do
+
```

Comment:
Don’t forget to fill this empty test with content!

Reply:
will do :)